internal/observability/call.go

```diff
@@ -222,7 +221,7 @@ func (c call) endLogs(
 		return
 	}
 
-	fields := c.fields[:0]
+	fields := make([]zapcore.Field, 0, 10)
```

Comment:
This is variadic function that accept arbitrary number of filed in `extraLogFields` argument (it works now somehow, because there are no calls with more than one field in extraLogFields, I guess).

So I would say it's better to use smth like:
`fields := make([]zapcore.Field, 0, 9+len(extraLogFields))`

Otherwise one day someone will add another field, and we'll have performance penalty here for expending slice (new allocation + copy).